login-service/src/api-login/auth-clients.controller.ts

```diff
@@ -156,7 +156,8 @@ export class AuthClientController {
         } else {
             newClient.isValid = true;
         }
-        if (input.requiresSecret != undefined) {
+        newClient.clientSecrets = []
```

Comment:
Is this list nonempty or null before, or why do you assign an empty list? Especially, since it is not filled in the remaining method.

Reply:
afaik it's undefined before